### PR TITLE
fix duration_ms to be correct

### PR DESCRIPTION
**Pull request by MoLow**
this fixes the same historical mistake from https://github.com/nodejs/node/pull/44450

### CODE REVIEW

File: tap2junit/__main__.py [hunk 1]
```diff
@@ -30,8 +30,8 @@ def extract_test_info_from_description(description):
 
 def map_yaml_to_junit(test):
     yaml = test.yaml or {}
-    # Even though the name is `duration_ms` the value is in seconds.
-    elapsed_sec = yaml.get("duration_ms", None)
+    duration_ms = yaml.get("duration_ms", None)
+    elapsed_sec = duration_ms if duration_ms is None else (duration_ms / 1000)
```
Comment (cclauss) [inline tap2junit/__main__.py:34]:
```suggestion
    elapsed_sec = yaml.get("duration_ms", 0) / 1000 or None
```

File: test/output/test2.xml [hunk 2]
```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="utf-8"?>
-<testsuites disabled="0" errors="0" failures="1" tests="1" time="1.711">
-	<testsuite disabled="0" errors="0" failures="1" name="test/fixtures/test2" skipped="0" tests="1" time="1.711" hostname="{HOSTNAME}">
-		<testcase name="test-accessor-properties" time="1.711000" classname="parallel">
+<testsuites disabled="0" errors="0" failures="1" tests="1" time="0.001711">
+	<testsuite disabled="0" errors="0" failures="1" name="test/fixtures/test2" skipped="0" tests="1" time="0.001711" hostname="{HOSTNAME}">
+		<testcase name="test-accessor-properties" time="0.001711" classname="parallel">
 			<failure type="1" message="fail (1)">
 ---
 duration_ms: 1.711
```
[the remark this answers is not in the document]
Comment (MoLow) [reply test/output/test2.xml:7]: this includes raw TAP - so it is expected to be milliseconds, but jUnit `time` attribute is expected as seconds

[the remark this answers is not in the document]
Comment (cclauss) [reply test/output/test2.xml:7]: Yes.  No wonder that everyone is confused.

**Comment by cclauss** [comment]
Changed lines `+4,757 −4,758`

**Review by aduh95 — APPROVED** [review]
I don't know enough about JUnit `time` attribute, but I trust that it's indeed in seconds, and the code changes LGTM.

**Comment by MoLow** [comment]
I think only @cclauss can merge this?

**Review by benjamingr — APPROVED** [review]

**Comment by targos** [comment]
Members of @nodejs/build are also admins of this repo

**Comment by rvagg** [comment]
added some comments about this @ https://github.com/nodejs/node/pull/44450#issuecomment-1238716561

**Pull request merged**